apps/parse-rated-paragraphs/src/app/parse-rated.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+
+const allParagraphsBasePath = './tmp/correlated-paragraphs/';
+function getAllJsonPaths(): string[] {
+  return fs.readdirSync(allParagraphsBasePath);
+}
+
+function getParagraphsFromFile(
+  filename: string
+): Promise<ParsedArticleParagraphStandalone[]> {
+  return new Promise((res, rej) => {
```

Comment:
why wrap this in a promise?

Reply:
I think fs.read only takes in a callback function, thus I wrapped it in a promise so that I could use it with async await

Reply:
I think that might overcomplicate the code a bit, since this doesn't actually need to be a promise. You should be able to make a regular javascript function and pass that in as a callback function without having to create a promise.
Either by doing: 
`function myfunc() { ... };
fs.read(myfunc) // note no ()s in this call means that you are passing the whole func as a param`
You can read more here: https://developer.mozilla.org/en-US/docs/Glossary/Callback_function

Reply:
Totally agree. I thought that callbacks may also create a more complicated structure. I found an elegant solution to a similar problem on [Stackoverflow](https://stackoverflow.com/questions/40593875/using-filesystem-in-node-js-with-async-await) though using `util.promisify`. Would this work?

Reply:
I think my question here is still why does it need to be a promise? When functions require callbacks I believe the recommended approach is using one of the two callback methods I mentioned above. We generally prefer not to make something a promise unless it needs to be

Reply:
Oh ok, I thought that promises were better style, but I can change